src/main.js

```diff
@@ -26,15 +28,26 @@ const getThisRelativePeriodRange = (unit, base = new Date(), num = 1) => {
   return formatRange(startOf(sub(dayBefore, num - 1)), formatDate(dayBefore));
 };
 
-// :: (Date | Int | String) -> Option(Date | Int | String) = new Date() -> Object
-const getTillYesterdayRange = (start, base = new Date()) => {
+const getTillYesterdayRange = (start: DateLike, base: Date = new Date()): Range => {
```

Comment:
Add `?` to `base` as it is optional?

And should `base` be `DateLike` instead?